pyfar/dsp/dsp_interpolation.py

```diff
@@ -0,0 +1,731 @@
+import numpy as np
+from scipy.special import iv as bessel_first_mod
+from scipy.interpolate import interp1d
+import scipy.signal as sgn
+import matplotlib.pyplot as plt
+import pyfar as pf
+
+
+def smooth_fractional_octave(signal, width, mode="magnitude",
+                             window="boxcar"):
+    """
+    Smooth spectrum with a fractional octave width.
+
+    The smoothing is done according to Tylka et al. 2017 [#]_ method 2 in three
+    steps:
+
+    1. Interpolate the spectrum to a logarithmically spaced frequency scale
+    2. Smooth the spectrum by convolution with a smoothing window
+    3. Interpolate the spectrum to the original linear frequency scale
+
+    Note: Method 3 in Tylka at al. 2017 is mathematically more elegant at the
+    price of a largely increased computational and memory cost. In most
+    practical cases, methods 2 and 3 yield close to identical results (cf. Fig.
+    2 and 3 in Tylka et al. 2017). If the spectrum contains extreme
+    discontinuities, however, method 3 is superior (see examples below).
```

Comment:
What about moving that to ` Notes` before the examples?

Reply:
good idea!